src/app/components/crypto-assets/crypto-currency-asset/crypto-currency-asset-item.layout.tsx

```diff
@@ -46,9 +69,9 @@ export const CryptoCurrencyAssetItemLayout = forwardRefWithAs(
         {...rest}
         {...bind}
       >
-        <Flag img={icon} align="middle" width="100%">
+        <Flag img={isHovered ? AssetItemCopyIcon(hasCopied) : icon} align="middle" width="100%">
```

Comment:
This should be JSX really rather than a function, `AssetItemCopyIcon` should accept `{ hasCopied: boolean }` not a single bool param

Reply:
now copy icon is passed as prop